provisioner/plain-v0/README.md

```diff
@@ -1,3 +1,197 @@
-# plain-v0
+# Plain-v0 Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The `plain-v0` provisioner is a core rukpak provisioner that knows how to interact with `plain+v0` format bundles.
```

Comment:
It may help to have a link here to https://github.com/operator-framework/rukpak/blob/main/README.md#provisionerclass for what the `rukpak provisioner` is.

Reply:
I agree we could link back to the README, but we will remove the ProvisionerClass from the top-level README at the moment since we have not implemented that API yet